src/main/java/carracing/ResultView.java

```diff
@@ -0,0 +1,29 @@
+package carracing;
+
+import java.util.List;
+
+public class ResultView {
+    private final StringBuilder output;
+    private static final String CAR_MOVESTATUS_DASH = "-";
+    private static final String NEW_LINE = "\n";
```

Comment:
[컨벤션](https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html)에 따르면 클래스 변수가 인스턴스 변수보다 먼저 작성되어야 합니다.
개행으로 두 변수들을 분리하면 가독성에도 도움이 될 것 같아요

개행 문자는 자바에서 제공하는 상수를 추천드립니다
```suggestion
    private static final String CAR_MOVESTATUS_DASH = "-";
    private static final String NEW_LINE = System.lineSeparator();

    private final StringBuilder output;
```